src/components/DashboardFilters/DashboardFilters-container.tsx

```diff
@@ -139,9 +139,8 @@ const Filters = (props: any) => {
 
   const fetchNewData = (activeBlockchains: string[]) => {
     activeBlockchains.forEach((blockchain: string) => {
-      console.log(filters);
```

Comment:
Why not just call `actions.fetchWalletsByDataSource` with parameter `blockchain` like `actions.fetchWalletsByDataSource({limit: filters.limit, from: filters.from, to: filters.to}, blockchain)`?

You should never NOT use variables in cases where the same value is used in multiple places, unless you want to get intimate with buggy and bad code.

Declare some const enums or variables e.g. 
```
enum Blockchains {
  ETH = 'Ethereum',
  TZX = 'Tezos'
}
```